scripts/report/_01_make_reports.py

```diff
@@ -232,7 +233,7 @@ def _plot_time_by_time_decoding_scores(
         max_scores = np.diag(max_scores)
         min_scores = np.diag(min_scores)
 
-    fig, ax = plt.subplots()
+    fig, ax = plt.subplots(constrained_layout=True)
```

Comment:
I want to change all of these (see https://github.com/mne-tools/mne-bids-pipeline/issues/602#issuecomment-1243836602). I can do it in a separate PR if it would help, but it seemed easy enough to do it here